src/patterns/InPlaceFilters/_inPlaceFilters.scss

```diff
@@ -0,0 +1,160 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-in-place-filters {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $spacing-sm;
+}
+
+.#{$px}-filter-button {
```

Comment:
should move this to a `_filterButton.scss` file and import directly in the component so they can be tested in isolation